compiler/passes/normalize.cpp

```diff
@@ -1179,6 +1184,9 @@ static void normalizeVariableDefinition(DefExpr* defExpr) {
       if (type == NULL) {
         init_untyped_var(var, init, stmt, constTemp);
 
+      } else if (init == NULL) {
+        init_typed_var(var, type,       stmt, constTemp);
```

Comment:
I think I get what you're trying to do here, but the insertion of blank space will probably be confusing for a casual reader.  Instead, could you maybe have a small comment describing what the space represents, i.e.

`init_typed_var(var, type, /* init=NULL, */ stmt, constTemp); // skips the init argument`

Otherwise, I think a code reader at a later point will be confused why the space is there and just remove it.